crates/bevy_ecs/src/query/fetch.rs

```diff
@@ -329,9 +327,9 @@ pub unsafe trait WorldQuery: for<'w> WorldQueryGats<'w> {
 
     /// Returns true if (and only if) every table of every archetype matched by this fetch contains
     /// all of the matched components. This is used to select a more efficient "table iterator"
-    /// for "dense" queries. If this returns true, [`WorldQuery::set_table`] and [`WorldQuery::table_fetch`]
-    /// will be called for iterators. If this returns false, [`WorldQuery::set_archetype`] and
-    /// [`WorldQuery::archetype_fetch`] will be called for iterators.
+    /// for "dense" queries. If this returns true, [`WorldQuery::set_table`] before [`WorldQuery::fetch`]
```

Comment:
```suggestion
    /// for "dense" queries. If this returns true, [`WorldQuery::set_table`] will be used before [`WorldQuery::fetch`]
```
Nit: seems like you forgot a few words.